plugins/py3embed/py3embed.cpp

```diff
@@ -131,6 +136,14 @@ static void failx(const char *message, ...)
     StringBuffer msg;
     msg.append("pyembed: ").valist_appendf(message,args);
     va_end(args);
+#ifdef HAS_BACKTRACE
+    void *stack[5];
+    unsigned nFrames = backtrace(stack, 5);
+    char** strs = backtrace_symbols(stack, nFrames);
+    for (unsigned i = 0; i < nFrames; ++i)
+        msg.append("\n  ").append(strs[i]);
```

Comment:
Will the newlines mess up the logging?  Would it be better to have them all on the same line?

Reply:
It's very ugly on one line. This string is not logged unless the ECL code outputs it, AFAIK